docs/docs/quick-start.md

```diff
@@ -13,43 +13,45 @@ title: ઝડપી શરૂઆત
 
 **નોંધ**: આ વિડિઓ માં `npx` નો ઉપયોગ  થયો છે, જે npm પેકેજને પ્રથમ ઇન્સ્ટોલ કર્યા વિના ચલાવવાનું એક સાધન છે. તમારા કમ્પ્યુટર પર Gatsby-CLI સ્થાપિત કર્યા પછી કમાન્ડ `npx gatsby new` ચલાવો એ `gatsby new` ચલાવવા જેવું જ છે.
 
-### Gatsby CLI ઇન્સટોલ કરો.
+### Gatsby CLI ઇન્સટોલ કરો
 
 ```shell
 npm install -g gatsby-cli
 ```
 
-### નવી સાઇટ બનાવો.
+> The above command installs Gatsby CLI globally on your machine.
```

Comment:
Please translate this line.

Reply:
I have translated requested changes and mark it as resolve.